src/firefly/js/ui/dynamic/EmbeddedPositionSearchPanel.jsx

```diff
@@ -212,6 +216,8 @@ EmbeddedPositionSearchPanel.propTypes= {
     searchItem: object,
     initArgs: object,
     slotProps: shape({
-        searchRoot: object
+        searchRoot: object,
+        searchInnerLayout: object,
+        searchTypeCone: object
```

Comment:
The current code is not slots friendly. Therefore, I see several issues with this slotProps declaration. However, refactoring it to utilize slotProps is beyond the scope of this PR. I would suggest adding a comment to indicate that a complete refactor is acceptable and not to consider these slots as absolutely required.